web/packages/extension/tools/sign_xpi.js

```diff
@@ -54,4 +55,8 @@ async function sign(
             "Skipping signing of Firefox extension. To enable this, please provide MOZILLA_API_KEY, MOZILLA_API_SECRET and FIREFOX_EXTENSION_ID environment variables"
         );
     }
-})();
+})().catch((error) => {
+    console.error("Error while signing Firefox extension:");
```

Comment:
`console.log` logs to stdout whereas `console.error` logs to stderr.
It's a bit weird to log these to separate streams.
How about logging both to stderr?
```js
console.error(`Error while signing Firefox extension: ${error}`);
```

Reply:
Oops, you're right.
Two separate `console.error` calls because npm pretty-prints the object if the parameter is an Object, but passing it in the string template just prints out `object`.